ml_services/api/views.py

```
@@ -0,0 +1,30 @@
+from flask import Blueprint, request, jsonify, current_app
+from ml_services.api.ml_utils import predict_score
+import os
+
+# Create a Blueprint
+base_blueprint = Blueprint('base', __name__, cli_group=None)
+
+@base_blueprint.route('/')
+def root():
+    return {'ok': True}
+
+@base_blueprint.route('/predict_score', methods=['POST'])
+def predict_score_route():
+    print("Predicting the score")
+    data = request.get_json()
+    message = data.get('message')
+    # model_path = data.get('model_path')
+    model_path = current_app.config.get('TORCH_MODEL_PATH')
+
+    if not message:
+        return jsonify({'error': 'Invalid input'}), 400
+
+    try:
+        if not model_path or not os.path.exists(model_path):
```

is this what we had before?

i think it should have a benign response if model_path isn't set (and a comment explaining this, probably log.info as well).

but if the model_path is set and not found, it should raise an exception / return error and log such.